client/components/web-preview/index.jsx

```diff
@@ -96,7 +61,11 @@ const WebPreview = React.createClass( {
 			document.documentElement.classList.add( 'no-scroll', 'is-previewing' );
 		}
 		this.props.setPreviewShowing( this.props.showPreview );
-	},
+	}
+
+	shouldComponentUpdate( nextProps, nextState ) {
+		return shallowCompare( this, nextProps, nextState );
```

Comment:
Good point about connect. I was aware of that but didn't think of it.

Also, I didn't realize we had a `PureComponent` in use. I will try and provide an update here to take this out (happily so)

Thanks!